### PR TITLE
add ssh-copy-id instructions

### PR DESCRIPTION
**Pull request by tyleretters**

### CODE REVIEW

File: norns/wifi-files.md [hunk 1]
```diff
@@ -173,6 +173,54 @@ When connected via WiFi you can SSH into norns from another computer on the same
...
+  Hostname norns.local
+  User we
+  
+Host *
+  AddKeysToAgent yes
+  UseKeychain yes
+  IdentityFile ~/.ssh/id_rsa
+```
+
+The `Host *` section assumes macOS and that one followed the Github instructions above. This configures `ssh` to use the `~/.ssh/id_rsa` for all connections. Change `id_ras` if you used a different name.
```
Comment (tehn) [inline norns/wifi-files.md]: `id_ras` typo

File: norns/wifi-files.md [hunk 2]
```diff
@@ -173,6 +173,54 @@ When connected via WiFi you can SSH into norns from another computer on the same
...
+
+Restart your terminal for the changs to take affect. Now run `ssh-copy-id norns`. YOu should see output similar to this and be prompted for a password:
```
Comment (tehn) [inline norns/wifi-files.md]: YOu

**Review by tehn — COMMENTED** [review]
rad, thanks!

not sure if the extra instructions just make it intimidating, but all i ever do (provided i have an ssh key generated already) is `ssh-copy-id norns.local`

and then when i want to login: `ssh we@norns.local`

substantially less setup for only slightly less convenience... so wondering if it makes sense to break the steps up?

**Comment by tyleretters** [comment]
@tehn where would you break the steps up? .ssh/config vs ssh-copy-id?

**Comment by tehn** [comment]
to me there are two parts:

1. copying the key and having `ssh we@norns.local` work w/o pw (to me, this is the main motivation)
2. the second part is registering the norns host and default username so we you can `ssh norns` which is slightly more obscure but also cool (i haven't bothered doing this myself)

**Comment by tyleretters** [comment]
yeah good call! updated. i think i got it all.

**Review by tehn — APPROVED** [review]

**Pull request merged**